cmake/installing.cmake

```diff
@@ -19,6 +19,13 @@ if (BUILD_WINDOWS_SHELL_THUMBNAILS_EXTENSION)
     RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT shellext)
 endif()
 
+# Java bindings
+if (F3D_JAVA_BINDINGS)
+  install(TARGETS javaf3d
+    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT java)
+  install_jar(f3d-jar DESTINATION "share/java" COMPONENT java)
```

Comment:
where is instal_jar coming from ?

Reply:
CMake `UseJava` module: https://cmake.org/cmake/help/latest/module/UseJava.html

Reply:
ok !